group_vars/all/dynamic.yml.example

```diff
@@ -0,0 +1,17 @@
+# Variables that will change between different contests.
+# Copy this file to `dynamic.yml` and change the variables
+
+# Password for the LDAP admin user
+ldap_admin_password: supersecretpassword
+
+# Organization to use for LDAP
+ldap_organization: ICPC
+
+# Domain name to use as base for LDAP in DNS format
+ldap_domain: icpc.global
+
+# Domain name in LDAP DC format. Should match ldap_domain
+ldap_domain_dc: dc=icpc,dc=global
+
+# IP address of the LDAP server
+ldap_server_ip: 10.0.0.203
```

Comment:
Just general remark: If there is only 1 ldapserver we can get the IP from that ansible_host

Reply:
Read my comment, I first had that but not sure if it is nicer? What do you think?

Reply:
You're aware of this.